### PR TITLE
BaSP-MR-109: subscriptions forms and validations

### PR DESCRIPTION
**Pull request by Dannylez**

### CODE REVIEW

File: src/Components/Subscriptions/Form/index.jsx [hunk 1]
```diff
@@ -77,13 +74,32 @@ const Form = () => {
...
+      name: `${obj?.activityId?.name}, ${obj?.day}, ${obj?.hour} hrs`
+    };
+  });
+
+  const selectMembers = members.map((obj) => {
+    return { _id: obj?._id, value: obj?._id, name: `${obj?.lastName}, ${obj?.firstName}` };
+  });
+
+  const onSubmit = async (data) => {
+    console.log(data);
```
Comment (DelfiLT) [inline src/Components/Subscriptions/Form/index.jsx:99]: Remove console.log

**Review by DelfiLT — COMMENTED** [review]
I can't subscribe to a class today and I can suscribe to a class next year

**Review by DelfiLT — APPROVED** [review]

**Review by SantiH97 — APPROVED** [review]

**Review by keiler-art — COMMENTED** [review]
It would be great if you could add a description to your PR!

**Comment by Dannylez** [comment]
[Subscriptions, Members & Activities - Implement React Hook Form](https://trello.com/c/bOcu53Yh/109-subscriptions-members-activities-implement-react-hook-form)

**Comment by ukiweskamp** [comment]
[Colors and Buttons - BUG](https://trello.com/c/wGE6lanS/83-colors-and-buttons-bug)

**Pull request merged**